Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -307,4 +299,7 @@ public static void DismissAlert(this UITestContext context)
 
     public static Task GoToContentItemEditorByIdAsync(this UITestContext context, string contentItemId) =>
         context.GoToAdminRelativeUrlAsync($"/Contents/ContentItems/{contentItemId}/Edit");
+
+    private static UriBuilder ReturnRelativeUri(this UITestContext context, string relativeUrl) =>
+        new(context.Scope.BaseUri) { Path = context.AdminUrlPrefix + relativeUrl, };
```

Comment:
Keep using `GetAbsoluteUri`.

Reply:
Now I think the solution looks clean.

Reply:
What's the advantage of your custom code compared to `GetAbsoluteUri` that's exactly for creating the absolute URL from a relative one?

Reply:
You are right I missed that Uri has the LocalPath which is equal to UriBuilder.Path.
I thought that I had to hack out the path if I return Uri.